starter-code/ShoppingListVer2/app/src/main/java/ly/generalassemb/drewmahrt/shoppinglistver2/DbHelper.java

```diff
@@ -0,0 +1,29 @@
+package ly.generalassemb.drewmahrt.shoppinglistver2;
+
+import android.content.Context;
+import android.database.sqlite.SQLiteDatabase;
+import android.database.sqlite.SQLiteOpenHelper;
+
+/**
+ * Created by ds on 10/25/16.
+ */
+
+public class DbHelper extends SQLiteOpenHelper {
```

Comment:
In this class we're missing a method to actually perform a query and return a list of shopping items. You have some of the logic for that in your MainActivity, but that should go in a separate method in this class instead.